product-suite/dual-investment-pools-dip/README.md

```diff
@@ -4,13 +4,13 @@ description: Improved DOVs
 
 # Dual Investment Pools (DIP)
 
-Dual Finance has upgraded decentralized option vaults (DOVs), by creating Dual Investment Pools (DIPs) offering streaming executable yield strategies. Customize strategies based on your risk appetite and earn yield immediately.\
+Dual Finance has advanced decentralized option vaults by creating Dual Investment Pools (DIPs), offering real-time yield strategies for your crypto assets. A variety of option strike prices and expiries are available and the premium ($USDC) and yield (APY%) are streamed live, allowing for greater transparency and ease of comparison across different DIPs. Users can customize strategies based on their own risk appetite and objectives.\
 \
-DIPs are also physically settled, meaning you will end up with either your staked asset or stablecoins upon withdrawing after expiration - the risk are these dual outcomes. Premiums are paid immediately upon staking, allowing users to put funds to work for longer. \
+DIPs are also physically settled, meaning that upon expiration and withdrawal either your staked asset or stablecoins will be returned to you depending on if the options expired in the money or out of the money. Premiums are paid immediately upon staking, increasing capital efficiency and allowing users to put the premium payment to work during the life of the option. \
```

Comment:
Replace with "in-the-money (ITM) or out-of-the-money (OTM)"

Reply:
Done!